queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- [ ] Optional suggestion

- You can modify the selected line with  `SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12'; `